package.json

```diff
@@ -18,6 +18,7 @@
         "@material-ui/icons": "^4.9.1",
         "@mdx-js/mdx": "^1.6.16",
         "gatsby": "^2.24.54",
+        "gatsby-plugin-prefetch-google-fonts": "^1.4.3",
         "gatsby-plugin-web-font-loader": "^1.0.4",
```

Comment:
Should this go away then?

Reply:
You mean web font loader?

Reply:
Yeah
I didn't fetch/compile but saw the file diff and wondering if the prefetch-google-fonts is a substitute for web-font-loader

Reply:
Yes it is, ill remove it in the next pr i am making for these changes